libmachine/provision/configure_swarm.go

```diff
@@ -70,6 +71,7 @@ func configureSwarm(p Provisioner, swarmOptions swarm.SwarmOptions, authOptions
 
 	swarmCmdContext := SwarmCommandContext{
 		ContainerName: "",
+		Env:           Env,
```

Comment:
As mentioned above, if you add the `Env` to `SwarmOptions` this can become `Env: swarmOptions.Env` instead and we don't have to add an additional argument to the function.